webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx

```diff
@@ -100,6 +100,7 @@ export const TemplatePlotsGrid: React.FC<TemplatePlotsGridProps> = ({
           <div
             key={plot}
             id={plot}
+            data-id={plot}
```

Comment:
Removing this gives a reliable failure on the flaky test (which can be fixed by updating the query in the listener).

Reply:
I wouldn't add this extra key. We don't have `data-id` anywhere else in the app and I don't really understand the reason to add it now.

Reply:
> I wouldn't add this extra key. We don't have data-id anywhere else in the app and I don't really understand the reason to add it now.

We're using it to target the plot's smooth range. Though If we want to avoid adding this, we should be able to use a HTML ref instead 🤔

Reply:
Using a HTML ref now!